apps/web/src/components/ui/password-form.tsx

```
@@ -0,0 +1,111 @@
+"use client";
+
+import * as React from "react";
+import { Eye, EyeOff } from "lucide-react";
+import { cn } from "@/lib/utils";
+
+export interface PasswordFormProps
+  extends Omit<React.InputHTMLAttributes<HTMLInputElement>, "type" | "size"> {
```

ESLint

[error] Replace `⏎··extends·Omit<React.InputHTMLAttributes<HTMLInputElement>,·"type"·|·"size"` with `·extends·Omit<⏎··React.InputHTMLAttributes<HTMLInputElement>,⏎··"type"·|·"size"⏎`

(prettier/prettier)